### PR TITLE
This fixes the  problem of requesting for iOS if the app is pending release

### PR DESCRIPTION
**Pull request by sfransen**
This fixes the  problem of requesting for iOS if the app is pending release

### CODE REVIEW

File: lib/codes/itunes_connect.rb [hunk 1]
```diff
@@ -1,85 +1,86 @@
 require 'fastlane_core/itunes_connect/itunes_connect'
 
 module Codes
-  class ItunesConnect < FastlaneCore::ItunesConnect
+	class ItunesConnect < FastlaneCore::ItunesConnect
 
-    PROMO_URL = "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/wa/LCAppPage/viewPromoCodes?adamId=[[app_id]]&platform=[[platform]]"
-    CODE_URL = "https://buy.itunes.apple.com/WebObjects/MZFinance.woa/wa/redeemLandingPage?code=[[code]]"
+	 # PROMO_URL = "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/wa/LCAppPage/viewPromoCodes?adamId=[[app_id]]&platform=[[platform]]"
```
Comment (KrauseFx) [inline lib/codes/itunes_connect.rb]: Can you remove the line that's commented out?

File: lib/codes/itunes_connect.rb [hunk 2]
```diff
@@ -1,85 +1,86 @@
...
+	PROMO_URL = "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/wa/LCAppPage/viewPromoCodes?adamId=[[app_id]]&versionString=latest&&platform=[[platform]]"
```
Comment (KrauseFx) [inline lib/codes/itunes_connect.rb]: There are 2 `&&` before the `platform`

File: lib/codes/itunes_connect.rb [hunk 3]
```diff
@@ -1,85 +1,86 @@
...
+			if app_id.to_i == 0 or app_identifier.to_s.length == 0
+				raise "Could not find app using the following information: #{args}. Maybe the app is not in the store. Pass the Apple ID of the app as well!".red
+			end
 
-      if app_id.to_i == 0 or app_identifier.to_s.length == 0
-        raise "Could not find app using the following information: #{args}. Maybe the app is not in the store. Pass the Apple ID of the app as well!".red
-      end
+		app = FastlaneCore::ItunesSearchApi.fetch(app_id)
+		platform = args[:platform]
+		Helper.log.info "platform using is #{platform}"
```
Comment (KrauseFx) [inline lib/codes/itunes_connect.rb]: Can you change that to `"Currently using platform '#{platform}'"

**Comment by KrauseFx** [comment]
Thanks @sfransen, I added some comments :+1:

**Comment by sfransen** [comment]
I made the changes you wanted done.

**Comment by KrauseFx** [comment]
@sfransen Thanks. I might see this wrong, but did you use a 4 spaces tab intend? For Ruby scripts it's standard to have 2 spaces only.

**Comment by sfransen** [comment]
Standard is 4 space instead for tab

I what just trying to line it all up as it was not lined up right

**Comment by KrauseFx** [comment]
@sfransen that's not correct, 2 spaces is standard for Ruby projects: https://stackoverflow.com/questions/6741586/rails-coding-standards-why-2-space-indentation

**Comment by sfransen** [comment]
Fine I will change tab to 2 spaces

**Pull request closed without merging**